docs/pages/router/reference/link-preview.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Link Preview
+description: Learn how to add a preview to your link on iOS.
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+
+<ContentSpotlight
+  alt="A video presenting the link preview feature in Expo Router."
+  file="/expo-router/link-preview.mp4"
+/>
+
+Link preview (also known as "Peek and Pop") is a feature commonly used on iOS to show users a preview popup of the screen for a link.
+This guide will show you how to add and customize a link preview for your app on iOS.
+
+If you have a link in your app, you can add a link preview to it by replacing the link's content with a `<Link.Trigger>` and adding the `<Link.Preview>` component to it. This will create a preview of the page that the link points to.
```

Comment:
```suggestion
If you have a link in your app, you can add a link preview to it by replacing the link's content with a [`Link.Trigger`](/versions/latest/sdk/router/#linktrigger) and adding the [`Link.Preview`](/versions/latest/sdk/router/#linkpreview) component to it. This will create a preview of the page that the link points to.
```

- Sometimes in this doc, the `<>` are not used. Let's be consistent about this.
- Lets add reference links for components.